src/js/components/PageHeaderActionsMenu.js

```diff
@@ -0,0 +1,62 @@
+import {Dropdown} from 'reactjs-components';
+import React from 'react';
+
+import Icon from './Icon';
+
+const getMenuItems = (children) => {
+  return [
+    {
+      className: 'hidden',
+      html: <Icon id="ellipsis-vertical" size="mini" />,
+      id: 'trigger'
+    },
+    ...React.Children.map(children, getDropdownItemFromComponent)
+  ];
+};
+
+const handleItemSelection = (item) => {
+  if (item.clickHandler) {
+    item.clickHandler();
```

Comment:
I'm not sure if I completely understand your reasoning, but I would argue that in case we extend the functionality we should also add new handlers not reuse the`clickHandler`.  Let's follow our best practices and rename the handler to `onClick`.